BREEZE.rst

```diff
@@ -474,214 +617,213 @@ Run pylint checks for all files:
      ./breeze --static-check-all-files pylint
 
 
-The ``license`` check is also run via separate script and separate docker image containing
+The ``license`` check is run via a separate script and a separate docker image containing the 
```

Comment:
```suggestion
The ``license`` check is run via a separate script and a separate Docker image containing the 
```

Reply:
Should we make proper names start with an upper case? I noticed that you did that nearly every time for Airflow.

I think we then should do it then for Docker, SQLite, etc. as well. Can you check everything again in case I missed some?

Reply:
Yep. We gradually fix it with @efedotova :).